static/app/views/issueDetails/streamline/sidebar/metricIssueSidebarSection.tsx

```diff
@@ -0,0 +1,26 @@
+import {LinkButton} from 'sentry/components/button';
+import {Flex} from 'sentry/components/container/flex';
+import {t} from 'sentry/locale';
+import type {Event} from 'sentry/types/event';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export function MetricIssueSidebarSection({event}: {event?: Event}) {
+  const organization = useOrganization();
+  const alert_rule_id = event?.contexts?.metric_alert?.alert_rule_id;
+
+  if (!alert_rule_id) {
+    return null;
+  }
```

Comment:
feels like we can skip the config and just allow it to display none

Reply:
trying to avoid having a million things in the issue type config

Reply:
ah good point